CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 ## StreamChat
+### ✅ Added
+- `quotesEnabled` property is added to the `ChannelConfig` [#1891](https://github.com/GetStream/stream-chat-swift/issues/1891)
```

Comment:
Can we add an entry to ## StreamChatUI as well? 🙏

Reply:
Sure, updated the CHANGELOG.